static/webpack-config.js

```diff
@@ -25,11 +26,31 @@ module.exports = function () {
     });
   }
 
+  const globalConfigPath = `./${jamboConfig.dirs.config}/global_config.json`;
+  let globalConfig = null;
+  if (fs.existsSync(globalConfigPath)) {
+    globalConfigRaw = fs.readFileSync(globalConfigPath, 'utf-8');
+    globalConfig = parse(globalConfigRaw);
+  }
+
+  const useJWT = globalConfig && globalConfig.useJWT
+  let jamboInjectedData = process.env.JAMBO_INJECTED_DATA || null;
+  jamboInjectedData = (jamboInjectedData && JSON.parse(jamboInjectedData));
+
+  const getCleanedJamboInjectedData =
+    require(`./${jamboConfig.dirs.output}/static/webpack/getCleanedJamboInjectedData.js`);
+
```

Comment:
process.env.JAMBO_INJECTED_DATA comes in as a string, so would it be possible to only do the parsing and restringifying if useJWT is true? maybe something like the below. Then I think we can also remove the ternary operator in the DefinePlugin config below

JSON.parse(null) just returns null
```js
let jamboInjectedData = process.env.JAMBO_INJECTED_DATA || null;
if (useJWT && jamboInjectedData) {
  const getCleanedJamboInjectedData =
    require(`./${jamboConfig.dirs.output}/static/webpack/getCleanedJamboInjectedData.js`);
  jamboInjectedData = JSON.parse(jamboInjectedData)
  jamboInjectedData = getCleanedJamboInjectedData(jamboInjectedData)
  jamboInjectedData = JSON.stringify(jamboInjectedData)
}
```

Reply:
I like that better. updated!